### ISSUE TITLE
feature request! slot by points

### ISSUE THREAD

**Issue by TZMCKP** [raithos/xwing#948]
Hello, is it possible to choose points amount as a dropdown/alternative to ship type? With the new amg rules im finding myself building lists where im very committed to certain pilot and seeing whats available to round out, if it were possble to "select 4 point ships" from the dropdown for example, i at least would benefit. Thanks so much for all you do. Dee yun is the worst

**Comment by raithos** [comment]
ill think about it but unlikely, that would be a fundimental reworking on how we populate the ship types and would break a lot of things

**Comment by Patschke** [comment]
The card browser is already allowing you to search for that. It's only one click away, and you'd need to toggle between the sort scheme anyways - so I don't see a huge benefit here. The card browser might not be fully updated to 2.5 (e.g. no load out points) and is a bit slow, but I'd be curious to get feedback on of if it fits your needs

**Comment by TZMCKP** [comment]
yeah thats awesome! I never realized it was there! thank you so much!
